### PR TITLE
KOGITO-1841: Added serviceUrl on DE table

### PR DESCRIPTION
**Pull request by Sara4994**
Hello! please review, this PR containing a minor enhancement to display serviceUrl on domain explorer table.

JIRA issue: https://issues.redhat.com/browse/KOGITO-1841

### CODE REVIEW

**Review by nmirasch — APPROVED** [review]
Looks good to me and works fine. Ready to merge after jenkins green build.

**Review by AjayJagan — APPROVED** [review]

**Review by cristianonicolai — APPROVED** [review]
looks fine, the only remark about the format on index.ts should be fixed in https://issues.redhat.com/browse/KOGITO-2502

**Comment by cristianonicolai** [comment]
jenkins please retest this

File: ui-packages/packages/management-console/server/MockData/types.js [hunk 1]
```diff
@@ -82,7 +82,8 @@ module.exports = typeDefs = gql`
     start: DateTime!
     end: DateTime
     lastUpdate: DateTime!
-    businessKey: String
+    businessKey: String,
```
Comment (jstastny-cz) [inline ui-packages/packages/management-console/server/MockData/types.js]: should the comma be there? on other lines it is not.

Comment (Sara4994) [reply ui-packages/packages/management-console/server/MockData/types.js]: removed

**Review by jstastny-cz — APPROVED** [review]

**Pull request merged**